src/plone/exportimport/utils/content/export_helpers.py

```diff
@@ -114,8 +114,11 @@ def fix_language(
     """Fix language information."""
     placeholder = settings.PLACEHOLDERS_LANGUAGE
     default_portal_language = config.site.language
-    token = item.get("language", {}).get("token", placeholder)
-    lang = placeholder if token == default_portal_language else token
+    if isinstance(item.get("language", {}), str):
+        lang = placeholder
```

Comment:
I think we only want to use the placeholder here if the language is "" or equals the portal default language. Otherwise we should keep the language that is set on the content. No?